engine/components/IgeInputComponent.js

```diff
@@ -544,7 +544,15 @@ var IgeInputComponent = IgeEventingClass.extend({
 
 			if (isChatInputHidden && !isModalOpen && !isMenuVisible) {
 				// if (ige.client.myPlayer && ige.client.myPlayer._stats.email) {
+				var gameData = ige.game && ige.game.data && ige.game.data.defaultData;
+				var player = ige.client.myPlayer;
+				if(gameData.allowVerifiedUserToChat && !player._stats.isUserVerified){
+					$('#message').attr('disabled', true);
+					$('#message').attr('placeholder', 'please log in to chat');
```

Comment:
If we are capitalizing the first letter of `You are muted` we should capitalize the first letter of `please log in to chat`